src/util/color.js

```diff
@@ -1,39 +1,41 @@
-const Color = require("color");
-
-/*
-    Color generation
-    ================
-    Each app has its own color in app_colors
-    The first 4 apps get the 4 default colors
-    The preceeding apps get a mix in the order of 1+0,2+0,2+1,3+0,3+1,3+2...
-*/
-var colors = [Color('#90CAF9'), Color('#FFE082'), Color('#EF9A9A'), Color('#A5D6A7')];
-let app_colors = {};
-
-function calculateNextColor() {
-  var color1 = colors[(colors.length >> 1)-1];
-  var color2 = colors[colors.length%(colors.length >> 1)];
-
-  // Now mix the two
-  var color = Color({
-    r: (color1.rgb().r+color2.rgb().r)/2,
-    g: (color1.rgb().g+color2.rgb().g)/2,
-    b: (color1.rgb().b+color2.rgb().b)/2
-  });
-
-  colors[Object.keys(app_colors).length] = color;
+const d3 = require("d3");
```

Comment:
Yeah, the colors are hardly final. I just picked a color set included with d3.js and didn't feel like bikeshedding over colors at the time. Feel free to find a set you like better.